rumqttc/src/client.rs

```diff
@@ -207,17 +211,21 @@ fn get_ack_req(publish: &Publish) -> Option<Request> {
     Some(ack)
 }
 
-/// `Client` to communicate with MQTT eventloop `Connection`.
+/// A synchronous client, communicates with MQTT `EventLoop`.
 ///
-/// Client is cloneable and can be used to synchronously Publish, Subscribe.
-/// Asynchronous channel handle can also be extracted if necessary
+/// This is cloneable and can be used to synchronously [`publish`](`Client::publish`) and
+/// [`subscribe`](`Client::subscribe`).
```

Comment:
```suggestion
/// This is cloneable and can be used to synchronously [`publish`](`AsyncClient::publish`),
/// [`subscribe`](`AsynClient::subscribe`) through the `EventLoop`/`Connection`, which is to be polled in parallel
/// by iterating over the object returned by [`Connection.iter()`](Connection::iter) in a separate thread.
///
/// **NOTE**: The `EventLoop`/`Connection` must be regularly polled(`.next()` in case of `Connection`) in order to 
/// send, receive and process packets from the broker, i.e. move ahead.
```